src/scripts/OSFramework/OSUI/Pattern/Sidebar/Sidebar.ts

```diff
@@ -187,7 +187,10 @@ namespace OSFramework.OSUI.Patterns.Sidebar {
 		private _overlayMouseDownCallback(_args: string, e: MouseEvent): void {
 			const targetElem = e.target as HTMLElement;
 			this._clickedOutsideElement = true;
-			if (targetElem.closest('.osui-sidebar__header') || targetElem.closest('.osui-sidebar__content')) {
+			if (
+				targetElem.closest('.osui-sidebar__header') ||
```

Comment:
@BenOsodrac Don't we have an Enum for these CSS classes, instead of having them hardcoded?

Reply:
Done!